.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
Back on topic, I believe testing against "stable"/"node"/LTS is an anti-pattern.

It means CI will unsolicitedly bump the major Node version whenever it becomes available without any previous test or knowledge from the authors. Major versions indicate breaking changes, and thus should be throughout tested (e.g. via a PR bumping the major version, like this one) in order to prevent breaking the build overnight.